lib/db.js

```diff
@@ -1671,6 +1672,35 @@ Db.prototype.unref = function() {
   this.s.topology.unref();
 };
 
+/**
+ * Create a new Change Stream, watching for new changes (insertions, updates, replacements, deletions, and invalidations) in this database. Will ignore all changes to system collections.
+ * @method
+ * @since 3.0.0
+ * @param {Array} [pipeline=null] An array of {@link https://docs.mongodb.com/manual/reference/operator/aggregation-pipeline/|aggregation pipeline stages} through which to pass change stream documents. This allows for filtering (using $match) and manipulating the change stream documents.
```

Comment:
I thought you wanted to stop documenting things as defaulting to `null`! 😄

Reply:
😞 I failed to meet my own standards